otter/test/rest/test_groups.py

```diff
@@ -152,8 +166,30 @@ def test_format_state_different_status(self, config_value):
             True,
             ScalingGroupStatus.ERROR,
             desired=10)
-        result = format_state_dict(state, active)
+        result = self.format(state, active)
         self.assertEqual(result['status'], 'ERROR')
+        self.assertEqual(result['errors'], [])
```

Comment:
Non-blocking, since this is just a test, but this should technically never happen, right?  All errors should have at least one reason?